Chart.lock

```diff
@@ -11,5 +11,8 @@ dependencies:
 - name: prometheus-pushgateway
   repository: https://prometheus-community.github.io/helm-charts
   version: 2.7.1
-digest: sha256:94a6285bec694dcf7093fecea298669752b293fa8d6237c6cb06033cdf7d0071
-generated: "2024-03-06T14:04:27.646537661-05:00"
+- name: fleet-scheduler
+  repository: https://metal-toolbox.github.io/fleet-scheduler
+  version: v0.1.0
```

Comment:
Is the helm chart really pre-release? If not we should call this 1.0.

Reply:
Currently the released version is v0.1.0. After this all works well ill release a new tag with v1.0.0